apps/web/components/SignUpModal.tsx

```diff
@@ -0,0 +1,53 @@
+import { Input } from "antd";
+import { useTranslation } from "react-i18next";
+import { IoIosCloseCircle } from "react-icons/io";
+
+import { PrimaryButton } from "ui";
+import {
+  Container,
+  ContentModal,
+  TitleContent,
+  Title,
+  SubTitle,
+  CloseButton,
+  DataField,
+} from "../styles/components/ModalStyles";
+
+interface ModalProps {
+  signUpModal: Boolean;
+  handleSignUpModalVisibility: () => void;
+}
+
+export default function SignUpModal({ signUpModal, handleSignUpModalVisibility }: ModalProps) {
```

Comment:
@MFelipeSilva why do we need these props? we can just create a component called Modal instead of creating a SignInModal, SignUpModal, etc ...